CHANGES.md

```diff
@@ -1,3 +1,19 @@
+## edge-19.8.7
+
+* CLI
+  * Added a global `--cluster-domain` flag to `linkerd install` to allow
```

Comment:
We don't have this one yet.

Reply:
Oh, whoops --- I misunderstood the PR description in #3278